src/main/java/com/github/vkorobkov/jfixturescmd/Prompt.java

```diff
@@ -4,15 +4,27 @@
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.Properties;
+
 @Slf4j
 final class Prompt {
+    private static final Properties properties = new Properties();
```

Comment:
We probably need a separate singleton class with static typed accessors(getter method) for every property from `src/main/resources/project.properties`